input/pagecontent/changes.md

```diff
@@ -1,6 +1,16 @@
+###  Release 1.0.0
+- Expected Publication date: 2025-24-01
+- Expected Publication status: Normative
+- Based on FHIR version: 4.0.1
+
+This change log documents the significant updates and resolutions implemented from version 1.0.0-ballot to 1.0.0.
+
+#### Changes in this version
+- Removed Must Support flag and associated Obligation extensions from location.identifier [FHIR-47105](https://jira.hl7.org/browse/FHIR-47105) and location.mode [FHIR-47104](https://jira.hl7.org/browse/FHIR-47104)
```

Comment:
The comment for this change should explicitly mention the affected profile.

Reply:
Capitalised the L in location so it's more obvious that refers to a profile. That seems to be consistent with style of previous similar changes

Reply:
FHIRPath needs it to be case sensitive, please change to "Location.identifier" and "Location.mode".